src/subgraph-client/subgraph-client.service.spec.ts

```
@@ -0,0 +1,74 @@
+jest.mock('graphql-request', () => ({
+  // gql: jest.fn((...args) => args[0]), // => below is changing array into string
+  gql: jest.fn((strings: TemplateStringsArray, ...values: any[]) =>
+    strings.reduce((acc, str, i) => acc + str + (values[i] ?? ''), ''),
+  ),
+  request: jest.fn(),
+}));
+
+describe('SubgraphClientService', () => {
+  let service: SubgraphClientService;
+  let module: TestingModule;
+  let configService: ConfigService<IConfig, true>;
```

ESLint

[error] 'configService' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)